src/services/EscapePodService.test.ts

```diff
@@ -0,0 +1,129 @@
+import sinon from "sinon";
+import { stubInterface } from "ts-sinon";
+import NotFoundException from "../exceptions/NotFoundException";
+import EscapePod  from "../entities/EscapePod";
+import getEscapePodMock from "../mocks/EscapePodMock";
+import IRepository from "../types/IRepository";
+import EscapePodService from "./EscapePodService";
+
+
+describe(
+  "EscapePodService", () => {
+    const sandbox = sinon.createSandbox();
+
+    afterEach(() => sandbox.restore());
+
+    /* GET REQUESTS */
+
+    /* Get By Id Success */
+
+    it(
+      "getById success", async () => {
+        // Given
+        const escapePodMock = getEscapePodMock();
+        const fakeRepo = stubInterface<IRepository<EscapePod>>();
+
+        // When
+
```

Comment:
Let's just avoid so many blank lines, I don't know why linter doesn't complain about this but just for the sake of styling